guides/contribution-guide.md

```diff
@@ -1,13 +1,25 @@
 # Guidelines for Contribution

```

Comment:
You will see in my commit that I tried to standardize this title in a couple places. FYI as to why I made those changes.

Reply:
awesome thank you so much 👍